### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by VedanthB**
- added cart page

### CODE REVIEW

**Comment by Roshannair24** [comment]
Looks good to merge

**Review by Roshannair24 — APPROVED** [review]

File: index.css [hunk 1]
```diff
@@ -14,6 +14,13 @@
 .bg-regal-blue-dark {
   background-color: #000022;
 }
+.text-regal-blue {
+  color: #2a2840;
+}
+
+.text-regal-blue-dark {
+  color: #000022;
+}
```
Comment (prathmeshjagtap) [inline index.css]: you can add CSS variables  here

File: pages/product-page/product-listing.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.divider {
+  border: 1px solid var(--regal-blue-dark);
+  width: 80%;
+  margin: auto;
+}
+.filters-sidebar-container {
+  height: 100%;
+  width: 20%;
+  /* border: 1px solid var(--regal-blue-dark); */
```
Comment (prathmeshjagtap) [inline pages/product-page/product-listing.css]: remove Unnecessary Comments

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]

File: index.css (hunk 1 above)
Comment (VedanthB) [reply index.css]: okay will make the changes

File: pages/product-page/product-listing.css (hunk 2 above)
Comment (VedanthB) [reply pages/product-page/product-listing.css]: okay, will make the changes

**Pull request merged**